src/ocamlorg_frontend/css/styles.css

```diff
@@ -96,42 +96,47 @@ body {
 }
 
 .code-preview {
-  font-family: Roboto Mono,ui-monospace,SFMono-Regular,Menlo,Monaco,Consolas,Liberation Mono,Courier New,monospace;
```

Comment:
would prefer unrelated formatting changes to appear in a different pull request in favor of keeping the diff small on this one

Reply:
Oops! Code formatter was enabled by accident - ignore.

Reply:
same problem again, code formatter changed the formatting of styles.css